openassessment/xblock/ui_mixins/mfe/mixin.py

```diff
@@ -77,13 +77,10 @@ def get_learner_data(self, data, suffix=""):  # pylint: disable=unused-argument
 
         # Allow jumping to a specific step, within our allowed steps
         if jump_step:
-            jumpable_steps = ("submission", "peer", "grades")
-            if jump_step not in jumpable_steps:
-                raise JsonHandlerError(404, f"Invalid jump to step: {jump_step}")
             if self._can_jump_to_step(workflow_step, jump_step):
                 serializer_context.update({"jump_to_step": suffix})
             else:
-                raise JsonHandlerError(400, f"Cannot jump to step: {jump_step}")
+                return None
```

Comment:
It may be worth the effort to add a log statement here to capture why we can't jump to the requested step; ie, record the invalid jump vs just not possible. This can help capture the information we had in the old code.